enaml/qt/q_popup_view.py

```diff
@@ -863,7 +867,12 @@ def _layoutPlainRect(self):
         offset = self._state.offset
         trial_pos = target_pos + offset - anchor_pos
         trial_geo = QRect(trial_pos, self.size())
-        geo = ensure_on_screen(self._get_screen(), trial_geo)
+        if IS_X11:
+            # X11 handles geometry as an absolute position in the desktop
+            # when there are multiple screens
```

Comment:
Could you expand as to why it breaks and why we hence choose to bypass the logic? A reference to the Qt documentation would be nice too.